src/main/java/com/google/netpcapanalysis/dao/MaliciousIPDaoImpl.java

```diff
@@ -73,4 +78,65 @@ else if (result.getBody().contains(REQUEST_LIMIT))
         return data;
     }
     
+    private String isMalicious(String ip)
+    {
+        HttpResponse<String> apiResult;
+        String result;
+
+        String searchDB = ipCache.searchMaliciousDB(ip);
+        if(searchDB.equalsIgnoreCase(Flagged.TRUE))
+        {
+            result = Flagged.TRUE;
+        }
+        else if(searchDB.equalsIgnoreCase(Flagged.FALSE)){
+            result = Flagged.FALSE;
+        }
+        else{
+            System.out.println("Else statement");
```

Comment:
Remove debug statements. If you want to retain this add java logger and do log.debug().